extensions/http-status-codes/src/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { ActionPanel, Action, Icon, List } from "@raycast/api";
+import React from "react";
+import http from "http";
+
+type Code = {
+  code: string;
+  description?: string;
+};
+
+export default function Command() {
+  const codeGroups = Object.entries(http.STATUS_CODES).reduce(
+    (groups: { [firstDigit: string]: Code[] }, [code, description]) => {
+      const group = groups[code[0]] || [];
+      group.push({ code, description });
+      groups[code[0]] = group;
+
+      return groups;
+    },
+    {}
+  );
+
+  return (
+    <List isLoading={false} searchBarPlaceholder="Filter by code or description...">
+      {Object.entries(codeGroups).map(([firstDigit, codes]) => (
+        <List.Section key={firstDigit} title={`${firstDigit}xx`} subtitle={getCodeGroupDescription(firstDigit)}>
+          {codes.map((code) => (
+            <List.Item
+              key={code.code}
+              title={code.code}
+              subtitle={code.description}
+              keywords={[code.description || ""]} // make subtitle searchable
+              icon={{
+                source: Icon.Dot,
+                tintColor: statusCodeToColor(code.code),
+              }}
+              actions={
+                <ActionPanel>
+                  <Action.OpenInBrowser title="Open in browser" url={getCodeDocsUrl(code.code)} />
+                  <Action.CopyToClipboard content={code.code} />
+                </ActionPanel>
+              }
+            />
+          ))}
+        </List.Section>
+      ))}
+    </List>
+  );
+}
+
+// Based on Steve Schoger's https://www.steveschoger.com/status-code-poster/
+function statusCodeToColor(status: string): string {
+  switch (status[0]) {
+    case "1":
+      return "#cff9fe";
+    case "2":
+      return "#d4f7ae";
+    case "3":
+      return "#cdc7ff";
+    case "4":
+      return "#f9f4b5";
+    case "5":
+      return "#fbb7b7";
+    default:
+      return "#ffffff";
+  }
+}
```

Comment:
Best to use our [built-in colors](https://developers.raycast.com/api-reference/user-interface/colors). 
```suggestion
function statusCodeToColor(status: string): Color {
  switch (status[0]) {
    case "1":
      return Color.Yellow;
    case "2":
      return Color.Green;
    case "3":
      return Color.Blue;
    case "4":
      return Color.Red;
    case "5":
      return Color.Orange;
    default:
      return Color.Magenta;
  }
}
```

Reply:
Thanks, great suggestion! This also tackles the dark/light mode issue :)